openapi-contracts/src/test/resources/domains/conversation/v1/messages/request/AppMessageCarouselRequestDto.json

```diff
@@ -0,0 +1,158 @@
+{
+  "carousel_message": {
+    "cards": [
+      {
+        "title": "title value",
+        "description": "description value",
+        "media_message": {
+          "url": "url value"
+        },
+        "height": "MEDIUM",
+        "choices": [
+          {
+            "text_message": {
+              "text": "This is a text message."
+            },
+            "postback_data": "postback_data text"
+          },
+          {
+            "call_message": {
+              "title": "title value",
+              "phone_number": "phone number value"
+            },
+            "postback_data": "postback_data call"
+          },
+          {
+            "location_message": {
+              "coordinates": {
+                "latitude": 47.627980,
+                "longitude": -2.822915
+              },
+              "title": "title value",
+              "label": "label value"
+            },
+            "postback_data": "postback_data location"
+          },
+          {
+            "url_message": {
+              "title": "title value",
+              "url": "an url value"
+            },
+            "postback_data": "postback_data url"
+          }
+        ]
+      }
+    ],
+    "choices": [
```

Comment:
Will check with conversation team, thank you for highlighting (can we imagine that... e.g. kakao will accept a mix ?)

Current json file purpose is to ensure 100% coverage about carousel content from unit tests; so test follow specs but no problem to align in case of feedback are highlighting a specs issue